recipes/simple-websocket-server/all/conanfile.py

```diff
@@ -27,10 +27,17 @@ def _source_subfolder(self):
         return "source_subfolder"
 
     def requirements(self):
+        # only version 2.0.2 upwards is able to build against asio 1.18.0 or higher
         if self.options.use_asio_standalone:
-            self.requires("asio/1.16.1")
+            if self.version <= "2.0.1":
+                self.requires("asio/1.16.1")
+            else:
+                self.requires("asio/1.18.0")
         else:
-            self.requires("boost/1.73.0")
+            if self.version <= "2.0.1":
+                self.requires("boost/1.73.0")
+            else:
+                self.requires("boost/1.74.0")
```

Comment:
What is the boost incompatibility?
It's the first recipe on cci that requires different boost versions for different package versions.

Reply:
Boost 1.74 is using asio 1.18